tools/slo-generator/slo_generator/backends/stackdriver.py

```diff
@@ -164,23 +150,42 @@ def exponential_distribution_cut(self, timestamp, window, **kwargs):
 
         return (good_event_count, bad_event_count)
 
-    def query(self, project_id, timestamp, window, filter):
+    def exponential_distribution_cut(self, *args, **kwargs):
+        """Alias for `distribution_cut` method to allow for backwards
+        compatibility.
+        """
+        warnings.warn(
+            f'exponential_distribution_cut will be deprecated in version 1.0, '
```

Comment:
This is version 1.0. Should this be changed to 2.0?

Reply:
Yes, thanks :)